helm/coco-diamond/templates/daemonset.yaml

```diff
@@ -17,6 +17,9 @@ spec:
         app: {{ .Values.service.name }} 
```

Comment:
Let's see how we go about this one, as I disabled the docker stats in the one from the other branch, and we should keep that version in k8s